package.json

```diff
@@ -12,12 +12,19 @@
     "addresses",
     "dist/artifacts/*"
   ],
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "scripts": {
     "test": "forge test",
     "generate-merkle-test-file": "node scripts/merkle-test.mjs",
     "clean": "rm -rf ./dist/",
+    "prepack": "node js-scripts/copy-latest-deployment-addresses.mjs && yarn run wagmi && yarn build-ts && yarn copy-addresses",
+    "deploy": "node scripts/deploy.mjs",
     "coverage": "forge coverage --report lcov",
     "build": "forge build",
+    "build-ts": "tsup index.ts --format cjs --dts --sourcemap",
+    "copy-addresses": "cp -r addresses/ dist/addresses",
```

Comment:
we should copy over `addresses` and include `chainConfigs` as well.

Reply:
done